### DISCUSSION TITLE
Sketchware pro UPDATED?

### DISCUSSION THREAD

**Discussion by alambd34** [Sketchware-Pro/Sketchware-Pro#827] [General]
![Screenshot_2023-04-04-02-10-03-88_572064f74bd5f9fa804b05334aa4f912](https://user-images.githubusercontent.com/64100157/229616431-7d14ab57-73f8-4325-93ea-2027be52ba0a.jpg)

**Comment by argieboy** [comment]
Link please 😍

**Comment by kuskebabi** [comment]
no, its a rickroll link. that was arpil fools joke.
